package.json

```diff
@@ -22,18 +28,46 @@
   ],
   "author": "Andrew Clark <acdlite@me.com>",
   "license": "MIT",
+  "dependencies": {
+    "flux-standard-action": "^2.0.3",
+    "is-promise": "^2.1.0"
+  },
   "devDependencies": {
-    "babel": "^5.6.14",
-    "babel-core": "^5.6.15",
-    "babel-eslint": "^5.0.0",
-    "chai": "^3.0.0",
-    "chai-as-promised": "^5.1.0",
-    "eslint": "^0.24.0",
-    "eslint-config-airbnb": "0.0.6",
-    "mocha": "^2.2.5",
-    "sinon": "^1.15.4"
+    "@babel/cli": "^7.0.0-beta.46",
+    "@babel/core": "^7.0.0-beta.46",
+    "@babel/preset-env": "^7.0.0-beta.46",
+    "babel-core": "^7.0.0-0",
+    "babel-jest": "^22.4.3",
+    "eslint-config-jest-files": "^0.1.3",
+    "eslint-config-unicorn-camelcase": "^0.1.1",
+    "husky": "^0.14.3",
+    "jest": "^22.4.3",
+    "prettier": "^1.12.1",
+    "pretty-quick": "^1.4.1",
+    "regenerator-runtime": "^0.11.1",
```

Comment:
For what is this needed?

Reply:
babel 7: https://facebook.github.io/jest/docs/en/getting-started.html#using-babel

Reply:
Can you try without it?

> Note: Explicitly installing regenerator-runtime is not needed if you use npm 3 or 4 or Yarn

Reply:
Done